PrimaryInstallScript.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+
+ping -q -c1 google.com > /dev/null
+ 
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#install Epsilon-Hermes
+git clone https://github.com/UCSolarCarTeam/Epsilon-Hermes.git /opt/
+(cd /opt/Epsilon-Hermes && cd ../ && mv ./Epsilon-Hermes ./src && mkdir Epsilon-Hermes && mv ./src ./Epsilon-Hermes/)
+qmake /opt/Epsilon-Hermes/
+make /opt/Epsilon-Hermes/
+/opt/build/SchulichEpsilonHermes
+git clone https://github.com/alanxz/SimpleAmqpClient /tmp/SimpleAmqpClient/
+mkdir /tmp/SimpleAmqpClient/build
+(cd /tmp/SimpleAmqpClient/build && cmake -DRabbitmqc_INCLUDE_DIR=../../rabbitmq-c/librabbitmq -DRabbitmqc_LIBRARY=../../rabbitmq-c/build/librabbitmq ..)
+(cd /tmp/SimpleAmqpClient/build make)
+sudo mkdir /usr/local/include/SimpleAmqpClient
+sudo cp *.so* /usr/local/lib/
+sudo cp ../src/SimpleAmqpClient/*.h /usr/local/include/SimpleAmqpClient
+
+#install BackupCamera
+git clone https://github.com/UCSolarCarTeam/BackupCamera.git /opt/
```

Comment:
Doesn't it need to be compiled?

Reply:
correct me if I am wrong but I believe the scripts take care of it, the backup camera successfully functioned on the pi after running scripts